src/analysis/ptyp_of_type.ml

```diff
@@ -23,7 +23,9 @@ let rec module_type =
       | Unit -> Parsetree.Unit
       | Named (id, type_in) ->
         Parsetree.Named
-          (Location.mknoloc (Option.map ~f:Ident.name id), module_type type_in)
+          ( Location.mknoloc (Option.map ~f:Ident.name id),
+            module_type type_in,
+            [] )
```

Comment:
@riaqn is it correct to pass an empty mode list here?

Reply:
(Same as above)